inference-engine/src/vpu/custom_kernels/correlate.cl

```diff
@@ -154,292 +145,305 @@ void __attribute__((noinline)) crosscorrh(__private const half* restrict line0,
+                        for (c = c * 8; c < cur_subchannels; c++) {
+                            sum += line0[x1 * max_channels + c] * line1[(x1 + o) * max_channels + c];
                         }
 
-                        dline[blockIdx_x + (((o + neighborhood_grid_radius*stride2)/stride2)*topwidth)] += sum;
+                        dline[blockIdx_x + (((o + neighborhood_grid_radius * stride2) / stride2) * topwidth)] += sum;
                     }
                 }
             }
         }
     }
 }
 
-
-__kernel void correlate2_half(__global const half* restrict bottom0,
-                              __global const half* restrict bottom1,
-                              __global half* restrict top,
-                             int topwidth,
-                             int topheight,
-                             int bottomwidth,
-                             int bottomheight,
-                             int bottomchannels,
-                             int max_displacement,
-                             int padding,
-                             int neighborhood_grid_radius,
-                             int neighborhood_grid_width,
-                             int kernel_size,
-                             int stride1,
-                             int stride2)
+__kernel void correlate2_half(
+    __global const half *restrict bottom0,
+    __global const half *restrict bottom1,
+    __global half *restrict top,
+    int topwidth,
+    int topheight,
+    int bottomwidth,
+    int bottomheight,
+    int bottomchannels,
+    int max_displacement,
+    int padding,
+    int neighborhood_grid_radius,
+    int neighborhood_grid_width,
+    int kernel_size,
+    int stride1,
+    int stride2)
 {
-    int max_channels = (MAX_OPENCL_BUFF_SIZE/sizeof(half) - topwidth*neighborhood_grid_width) / (3*bottomwidth);
+    int max_channels = (MAX_OPENCL_BUFF_SIZE / sizeof(half) - topwidth * neighborhood_grid_width) / (3 * bottomwidth);
     if (max_channels > 64) max_channels = 64;
     int subchannels_count = (bottomchannels + max_channels - 1) / max_channels;
-    int subchannels = (bottomchannels + subchannels_count-1) / subchannels_count;
+    int subchannels       = (bottomchannels + subchannels_count - 1) / subchannels_count;
     if (subchannels < max_channels) subchannels = max_channels;
 
-    const int sumelems = kernel_size*kernel_size*bottomchannels;
+    const int sumelems = kernel_size * kernel_size * bottomchannels;
 
-    __private half cmx[MAX_OPENCL_BUFF_SIZE/sizeof(half)];
+    __private half cmx[MAX_OPENCL_BUFF_SIZE / sizeof(half)];
 
-    __private half* line0 = cmx;
-    __private half* line1 = line0 + bottomwidth*subchannels;
-    __private half* dline = line1 + bottomwidth*subchannels;
+    __private half *line0 = cmx;
+    __private half *line1 = line0 + bottomwidth * subchannels;
+    __private half *dline = line1 + bottomwidth * subchannels;
 
     int blockIdx_y = get_global_id(0);
 
 #if defined(USE_MANUAL_DMA)
```

Comment:
could you please use the only one macro instead of two `USE_MANUAL_DMA` and `USE_DMA`